pylab/website/static/css/main.scss

```diff
@@ -38,3 +38,8 @@
         margin-top: 20px;
     }
 }
+.logo {
```

Comment:
You should refer to css class when you want to add staling to many elements. In this case, logo is the only element in the website, so you should refer to it via id.